packages/module/src/ChatbotConversationHistoryNav/ChatbotConversationHistoryDropdown.tsx

```diff
@@ -0,0 +1,61 @@
+// ============================================================================
+// Chatbot Header - Chatbot Conversation History Nav
+// ============================================================================
+import React from 'react';
+
+// Import PatternFly components
+import { MenuToggleElement, Tooltip, MenuToggle, Dropdown } from '@patternfly/react-core';
+
+import EllipsisIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+
+export interface ChatbotConversationHistoryDropdownProps {
```

Comment:
This should expand the DropdownProps (or something like that) so extra props can be spread to the PF Dropdown.